swarm.cabal

```diff
@@ -154,35 +155,26 @@ library swarm-lang
       -- See discussion in #415
       StrictData
 
-library swarm-engine
+library swarm-scenario
     import:           stan-config, common, ghc2021-extensions
     visibility:       public
     exposed-modules:  Swarm.Constant
-                      Swarm.Effect
-                      Swarm.Effect.Time
-                      Swarm.Game.Failure
-                      Swarm.Game.Achievement.Attainment
                       Swarm.Game.Achievement.Definitions
-                      Swarm.Game.Achievement.Description
-                      Swarm.Game.Achievement.Persistence
-                      Swarm.Game.CESK
                       Swarm.Game.Display
                       Swarm.Game.Entity
                       Swarm.Game.Entity.Cosmetic
                       Swarm.Game.Entity.Cosmetic.Assignment
-                      Swarm.Game.Exception
+                      Swarm.Game.Failure
```

Comment:
`Swarm.Game.Failure` would make more sense to me in `swarm-engine`.  Is there a reason it is needed here?

Reply:
Yes, scenario loading is still in the domain of the `swarm-scenario` sublibrary, though at some point we may revisit this.